### ISSUE TITLE
SC1010: 'then' words in Pacemaker-related scripts confuse ShellCheck

### ISSUE THREAD

**Issue by typeundefined** [koalaman/shellcheck#2266]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC1010
- My shellcheck version (`shellcheck --version` or "online"): 0.7.2 (fresh master)
- [ ] The rule's wiki page does not already cover this (e.g. https://shellcheck.net/wiki/SC1010)
- [x] I tried on https://www.shellcheck.net/ and verified that this is still a problem on the latest commit

#### Here's a snippet or screenshot that shows the problem:

Pacemaker command taken from the official [Red Hat resource](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/high_availability_add-on_reference/s1-orderconstraints-haar) fails

The script to reproduce the issue is as follows:
```sh
#!/bin/bash

pcs constraint order VirtualIP then dummy_resource kind=Optional
```

#### Here's what shellcheck currently says:
```
In ./pcs-test.sh line 3:
pcs constraint order VirtualIP then dummy_resource kind=Optional
                               ^-- SC1010: Use semicolon or linefeed before 'then' (or quote to make it literal).

For more information:
  https://www.shellcheck.net/wiki/SC1010 -- Use semicolon or linefeed before ...

```

#### Here's what I wanted or expected to see:
No warning. Reason: bash seems to process the script with no issue, so it sounds like a parser problem.
